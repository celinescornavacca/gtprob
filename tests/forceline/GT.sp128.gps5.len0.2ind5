((((((IB0,((IB3,IB4),(IB1,IB2))),(((JB3,JB4),JB0),(JB1,JB2))),(((WB0,WB1),(WB2,(WB3,WB4))),((AD0,(AD1,((AD3,AD4),AD2))),((BD0,(BD3,BD4)),(BD1,BD2))))),(((((((((VA0,(VA3,VA4)),(VA1,VA2)),((((OD3,OD4),((OD1,OD2),OD0)),(((PD3,PD4),(PD1,PD2)),PD0)),((YA1,YA2),((YA3,YA4),YA0)))),((((WA1,WA2),WA0),(WA3,WA4)),(XA0,(XA1,((XA3,XA4),XA2))))),((((QB1,(QB2,(QB3,QB4))),QB0),((RB0,(RB1,RB2)),(RB3,RB4))),(BB0,((BB3,BB4),(BB1,BB2))))),((FA1,(FA2,(FA3,FA4))),FA0)),(((XB0,(((XB3,XB4),XB2),XB1)),((((((WD1,WD2),WD0),(WD3,WD4)),((IE0,((IE1,IE2),(IE3,IE4))),((JE2,(JE3,JE4)),(JE0,JE1)))),((QD1,QD2),((QD3,QD4),QD0))),((CD1,(CD2,(CD3,CD4))),CD0))),((SA0,SA1),((SA3,SA4),SA2)))),((AA2,(AA3,AA4)),(AA0,AA1))),(((NA2,(NA3,NA4)),(NA0,NA1)),((OA0,OA1),((OA3,OA4),OA2))))),((((((PA0,((PA2,(PA3,PA4)),PA1)),(((QA2,(QA3,QA4)),QA1),QA0)),((ZA0,((ZA1,ZA2),(ZA3,ZA4))),((AB0,(AB1,AB2)),(AB3,AB4)))),((DA1,DA2),((DA3,DA4),DA0))),(((BA3,BA4),(BA1,BA2)),BA0)),(((XD0,(XD1,((XD3,XD4),XD2))),((YD0,(YD3,YD4)),(YD1,YD2))),((((EA0,(EA1,EA2)),(EA3,EA4)),((((GE1,GE2),GE0),(GE3,GE4)),((HE0,HE1),(HE2,(HE3,HE4))))),((CA0,CA1),(CA2,(CA3,CA4))))))),(((((((KA0,(((KA3,KA4),KA2),KA1)),(LA0,((LA1,LA2),(LA3,LA4)))),(((((((KE1,KE2),((KE3,KE4),KE0)),(((LE2,(LE3,LE4)),LE1),LE0)),(HB0,((HB2,(HB3,HB4)),HB1))),(EB0,((EB1,EB2),(EB3,EB4)))),((((UC0,(UC3,UC4)),(UC1,UC2)),((VC0,VC1),(VC2,(VC3,VC4)))),(PC0,(PC1,((PC3,PC4),PC2))))),(((IA1,(IA2,(IA3,IA4))),IA0),(((GD0,(GD3,GD4)),(GD1,GD2)),((EE0,((EE3,EE4),(EE1,EE2))),((FE0,FE1),(FE2,(FE3,FE4)))))))),(((((((UB1,UB2),UB0),(UB3,UB4)),((VB1,((VB3,VB4),VB2)),VB0)),((((EC0,((EC3,EC4),(EC1,EC2))),(((SC1,((SC3,SC4),SC2)),SC0),(TC0,(TC1,(TC2,(TC3,TC4)))))),(((XC2,(XC3,XC4)),(XC0,XC1)),((((SE3,SE4),(SE1,SE2)),SE0),(((TE3,TE4),(TE1,TE2)),TE0)))),(((PB3,PB4),PB0),(PB1,PB2)))),(((((UA1,UA2),UA0),(UA3,UA4)),((((((DC2,(DC3,DC4)),DC1),DC0),((((ZD0,(ZD3,ZD4)),(ZD1,ZD2)),(AE0,(AE1,(AE2,(AE3,AE4))))),(((DD2,(DD3,DD4)),DD1),DD0))),((((MD3,MD4),MD0),(MD1,MD2)),((ND1,ND2),(ND0,(ND3,ND4))))),((((((OB3,OB4),OB2),OB1),OB0),((((BC1,BC2),(BC3,BC4)),BC0),((CC1,CC2),(CC0,(CC3,CC4))))),((CB0,CB1),(CB2,(CB3,CB4)))))),((HA3,HA4),(HA0,(HA1,HA2))))),(((((UD2,(UD3,UD4)),UD1),UD0),(VD0,((VD1,VD2),(VD3,VD4)))),((FC3,FC4),(FC0,(FC1,FC2)))))),(((LC0,LC1),((LC3,LC4),LC2)),((MC1,MC2),(MC0,(MC3,MC4))))),((((((RD0,((RD1,RD2),(RD3,RD4))),(((SD2,(SD3,SD4)),SD1),SD0)),((WE0,(WE1,(WE2,(WE3,WE4)))),((XE3,XE4),(XE0,(XE1,XE2))))),((((((TA1,((TA3,TA4),TA2)),TA0),((((((DE1,DE2),DE0),(DE3,DE4)),(((UE3,UE4),(UE0,(UE1,UE2))),((VE0,(VE3,VE4)),(VE1,VE2)))),((JC3,JC4),(JC0,(JC1,JC2)))),(ZB0,((ZB3,ZB4),(ZB1,ZB2))))),((((ED0,(ED3,ED4)),(ED1,ED2)),((FD0,FD1),((FD3,FD4),FD2))),((((MB0,MB1),((MB3,MB4),MB2)),(((NB2,(NB3,NB4)),NB1),NB0)),(KB0,(KB1,(KB2,(KB3,KB4))))))),((((IC0,((IC3,IC4),(IC1,IC2))),((((OE3,OE4),OE2),(OE0,OE1)),(((PE1,PE2),(PE3,PE4)),PE0))),((((GC1,(GC2,(GC3,GC4))),GC0),((((JD3,JD4),JD2),(JD0,JD1)),(((KD3,KD4),((KD1,KD2),KD0)),((LD0,LD1),(LD2,(LD3,LD4)))))),((YB1,((YB3,YB4),YB2)),YB0))),(((QE0,(QE1,QE2)),(QE3,QE4)),((RE1,RE2),((RE3,RE4),RE0))))),((((NC0,(NC3,NC4)),(NC1,NC2)),(OC0,(((OC3,OC4),OC2),OC1))),(((GA1,GA2),(GA0,(GA3,GA4))),(((((((SB2,(SB3,SB4)),(SB0,SB1)),((TB1,TB2),(TB0,(TB3,TB4)))),(((YC0,((YC1,YC2),(YC3,YC4))),(((ZC3,ZC4),(ZC1,ZC2)),ZC0)),((KC0,KC1),((KC3,KC4),KC2)))),(DB0,(((DB3,DB4),DB2),DB1))),((TD0,(((TD3,TD4),TD2),TD1)),(((ME0,ME1),(ME2,(ME3,ME4))),(((NE3,NE4),(NE1,NE2)),NE0)))),((RA1,((RA3,RA4),RA2)),RA0)))))),((((FB2,(FB3,FB4)),(FB0,FB1)),(((GB2,(GB3,GB4)),GB1),GB0)),(((MA2,(MA3,MA4)),MA1),MA0))),(((((WC2,(WC3,WC4)),WC1),WC0),(((((BE3,BE4),BE2),BE1),BE0),(CE0,(((CE3,CE4),CE2),CE1)))),((AC1,(AC2,(AC3,AC4))),AC0)))),(((((HC3,HC4),(HC1,HC2)),HC0),((QC0,((QC3,QC4),(QC1,QC2))),(RC0,((RC2,(RC3,RC4)),RC1)))),(((JA1,((JA3,JA4),JA2)),JA0),(((HD0,(HD1,(HD2,(HD3,HD4)))),((ID1,((ID3,ID4),ID2)),ID0)),(LB0,((LB2,(LB3,LB4)),LB1)))))));
