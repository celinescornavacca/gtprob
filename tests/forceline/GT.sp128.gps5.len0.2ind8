((((((((((JD0,(JD1,JD2)),(JD3,JD4)),(KD0,(((KD3,KD4),KD2),KD1))),((DB0,(DB1,((DB3,DB4),DB2))),(EB0,(((EB3,EB4),EB2),EB1)))),(((RA0,((RA2,(RA3,RA4)),RA1)),(((((JB3,JB4),JB2),JB1),JB0),((KB0,KB1),((KB3,KB4),KB2)))),(LA0,((LA3,LA4),(LA1,LA2))))),(((((XB3,XB4),(XB1,XB2)),XB0),(((YB1,YB2),YB0),(YB3,YB4))),((((((GC3,GC4),(GC1,GC2)),GC0),((((DE1,((DE3,DE4),DE2)),DE0),((EE0,(EE3,EE4)),(EE1,EE2))),(ND0,((ND2,(ND3,ND4)),ND1)))),((((PA3,PA4),(PA0,(PA1,PA2))),((QA2,(QA3,QA4)),(QA0,QA1))),(JA0,((JA2,(JA3,JA4)),JA1)))),(((((GB3,GB4),GB0),(GB1,GB2)),(HB0,(HB1,((HB3,HB4),HB2)))),(((SE2,(SE3,SE4)),(SE0,SE1)),((TE1,TE2),((TE3,TE4),TE0))))))),(((BA0,((BA3,BA4),(BA1,BA2))),(((EA3,EA4),(EA0,(EA1,EA2))),(((ZA2,(ZA3,ZA4)),(ZA0,ZA1)),((((BB3,BB4),(BB1,BB2)),BB0),((CB3,CB4),(CB0,(CB1,CB2))))))),(((CA0,CA1),((CA3,CA4),CA2)),(((((((((UE0,((UE3,UE4),(UE1,UE2))),((VE1,(VE2,(VE3,VE4))),VE0)),((GD1,(GD2,(GD3,GD4))),GD0)),((CC0,(CC1,CC2)),(CC3,CC4))),((UB1,((UB3,UB4),UB2)),UB0)),((KA3,KA4),((KA1,KA2),KA0))),(((((((MC1,MC2),((MC3,MC4),MC0)),((((ME1,ME2),ME0),(ME3,ME4)),(((NE3,NE4),(NE1,NE2)),NE0))),((OB0,OB1),(OB2,(OB3,OB4)))),((IB0,IB1),((IB3,IB4),IB2))),(((VA0,(VA1,VA2)),(VA3,VA4)),(((AC2,(AC3,AC4)),(AC0,AC1)),((BC0,(BC1,BC2)),(BC3,BC4))))),((((IA3,IA4),IA2),IA1),IA0))),((WA0,(WA1,((WA3,WA4),WA2))),(((((JC0,(((JC3,JC4),JC2),JC1)),((KC1,((KC3,KC4),KC2)),KC0)),((FC0,(((FC3,FC4),FC2),FC1)),(((((KE1,KE2),(KE3,KE4)),KE0),((LE0,LE1),(LE2,(LE3,LE4)))),((((FE3,FE4),FE2),FE1),FE0)))),((((QC1,(QC2,(QC3,QC4))),QC0),((RC0,(RC1,RC2)),(RC3,RC4))),((TB2,(TB3,TB4)),(TB0,TB1)))),(((ED1,(ED2,(ED3,ED4))),ED0),((FD0,(FD3,FD4)),(FD1,FD2)))))),(((SA0,((SA3,SA4),(SA1,SA2))),(((((SD0,SD1),((SD3,SD4),SD2)),(TD0,(((TD3,TD4),TD2),TD1))),(ZB0,(((ZB3,ZB4),ZB2),ZB1))),(LB0,((LB3,LB4),(LB1,LB2))))),(((((IE0,(IE1,((IE3,IE4),IE2))),(JE0,(((JE3,JE4),JE2),JE1))),((LC0,LC1),((LC3,LC4),LC2))),((MB1,(MB2,(MB3,MB4))),MB0)),((TA0,(TA3,TA4)),(TA1,TA2)))))))),(((HA0,((HA2,(HA3,HA4)),HA1)),(((((HD1,HD2),(HD3,HD4)),HD0),(ID0,(((ID3,ID4),ID2),ID1))),((MA2,(MA3,MA4)),(MA0,MA1)))),(((((OD1,OD2),OD0),(OD3,OD4)),((PD0,(PD1,PD2)),(PD3,PD4))),(((OE1,((OE3,OE4),OE2)),OE0),(((PE3,PE4),PE2),(PE0,PE1)))))),(((AA1,AA2),AA0),(AA3,AA4))),((((DA0,DA1),((DA3,DA4),DA2)),(((QB3,QB4),(QB0,(QB1,QB2))),((((QE0,(QE1,((QE3,QE4),QE2))),(RE0,((RE1,RE2),(RE3,RE4)))),(YC0,(YC1,((YC3,YC4),YC2)))),((QD0,(((QD3,QD4),QD2),QD1)),(RD0,(RD1,((RD3,RD4),RD2))))))),((((((VD2,(VD3,VD4)),(VD0,VD1)),((WD0,((WD1,WD2),(WD3,WD4))),((XD0,XD1),(XD2,(XD3,XD4))))),(((FA3,FA4),(FA1,FA2)),FA0)),((((((LD0,(((LD3,LD4),LD2),LD1)),(MD0,(MD1,((MD3,MD4),MD2)))),(PC0,(((PC3,PC4),PC2),PC1))),(((((WC2,(WC3,WC4)),WC1),WC0),(XC0,(((XC3,XC4),XC2),XC1))),(SC0,((SC1,SC2),(SC3,SC4))))),(FB0,((FB1,FB2),(FB3,FB4)))),(((((CD0,(CD1,((CD3,CD4),CD2))),((DD0,(DD3,DD4)),(DD1,DD2))),((HC0,(HC1,(HC2,(HC3,HC4)))),(IC0,((IC1,IC2),(IC3,IC4))))),(((VB1,VB2),((VB3,VB4),VB0)),(WB0,(WB1,((WB3,WB4),WB2))))),(((GA1,GA2),(GA0,(GA3,GA4))),(((((NB2,(NB3,NB4)),(NB0,NB1)),(((((BD3,BD4),BD2),(BD0,BD1)),(((WE0,(WE1,WE2)),(WE3,WE4)),(((XE1,XE2),XE0),(XE3,XE4)))),((UC0,(UC1,((UC3,UC4),UC2))),((((VC3,VC4),VC2),VC1),VC0)))),((((((XA1,XA2),(XA0,(XA3,XA4))),(((YA1,YA2),YA0),(YA3,YA4))),(((RB3,RB4),(RB0,(RB1,RB2))),(SB0,((SB1,SB2),(SB3,SB4))))),((AB0,(AB1,((AB3,AB4),AB2))),(((UD0,UD1),((UD3,UD4),UD2)),((((BE0,(BE1,BE2)),(BE3,BE4)),(CE0,(CE1,((CE3,CE4),CE2)))),((AE0,AE1),(AE2,(AE3,AE4))))))),((((ZC0,ZC1),((ZC3,ZC4),ZC2)),(((AD3,AD4),AD0),(AD1,AD2))),(((GE0,(GE1,(GE2,(GE3,GE4)))),(HE0,(HE1,(HE2,(HE3,HE4))))),(((TC3,TC4),TC0),(TC1,TC2)))))),(((NA2,(NA3,NA4)),NA1),NA0)))))),((((OA1,OA2),((OA3,OA4),OA0)),(((DC0,((DC2,(DC3,DC4)),DC1)),(((EC2,(EC3,EC4)),EC1),EC0)),((UA1,UA2),((UA3,UA4),UA0)))),(((NC0,(NC1,(NC2,(NC3,NC4)))),((OC1,OC2),((OC3,OC4),OC0))),(((YD0,(((YD3,YD4),YD2),YD1)),(ZD0,(ZD1,((ZD3,ZD4),ZD2)))),(((PB2,(PB3,PB4)),PB1),PB0)))))));
