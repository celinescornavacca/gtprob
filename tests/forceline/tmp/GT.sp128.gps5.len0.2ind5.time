986.79user 1.85system 16:32.93elapsed 99%CPU (0avgtext+0avgdata 787200maxresident)k
121904inputs+8outputs (100major+689608minor)pagefaults 0swaps
