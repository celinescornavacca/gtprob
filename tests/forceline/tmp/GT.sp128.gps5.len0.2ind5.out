-1246.41142444
